mautic/mautic
# [UX] Put a placeholder on Alias and Public name when creating Segment to indicate that they’re auto-generated

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This enhancement introduces auto-generated placeholders for the ‘Alias’ and ‘Public Name’ fields in the Segment creation process. By indicating that these fields are automatically populated, we streamline the user experience, allowing users to focus on essential inputs without unnecessary concern. This change accelerates the workflow and simplifies the interface, fostering a more efficient and user-friendly environment.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Segments
3. Create new
4. Look at the fields

![Captura de tela 2024-02-28 135137](https://github.com/mautic/mautic/assets/116097999/b09c69b1-896c-448f-988d-dc14811f174e)
